python/cudf/cudf/core/groupby/groupby.py

```diff
@@ -1148,16 +1148,19 @@ def mult(df):
         ``engine='jit'`` may be used to accelerate certain functions,
         initially those that contain reductions and arithmetic operations
         between results of those reductions:
-        >>> import cudf
-        >>> df = cudf.DataFrame({'a':[1,1,2,2,3,3], 'b':[1,2,3,4,5,6]})
-        >>> df.groupby('a').apply(
-        ...     lambda group: group['b'].max() - group['b'].min(),
-        ...     engine='jit'
-        ... )
-        a  None
-        0  1     1
-        1  2     1
-        2  3     1
+
+        ..code-block:: python
+
+          import cudf
+          df = cudf.DataFrame({'a':[1,1,2,2,3,3], 'b':[1,2,3,4,5,6]})
+          df.groupby('a').apply(
+              lambda group: group['b'].max() - group['b'].min(),
+              engine='jit'
+          )
+          a  None
+          0  1     1
+          1  2     1
+          2  3     1
```

Comment:
After rendering, this format seems to be broken too, so we either can do this:

```suggestion

        >>> import cudf
        >>> df = cudf.DataFrame({'a':[1,1,2,2,3,3], 'b':[1,2,3,4,5,6]})
        >>> df.groupby('a').apply(
        ...   lambda group: group['b'].max() - group['b'].min(),
        ...   engine='jit'
        ... )
           a  None
        0  1     1
        1  2     1
        2  3     1
```

or this:

```suggestion

        .. code-block:: python
          import cudf
          df = cudf.DataFrame({'a':[1,1,2,2,3,3], 'b':[1,2,3,4,5,6]})
          df.groupby('a').apply(
              lambda group: group['b'].max() - group['b'].min(),
              engine='jit'
          )
          a  None
          0  1     1
          1  2     1
          2  3     1
```